src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java

```diff
@@ -52,6 +52,15 @@ public CantGetEditAssetsException(String message, Throwable cause) {
         }
     }
 
+    public static class Edit {
+        private Edit(ViewVisitor aVisitor, List<String> someUrls) {
```

Comment:
I saw this kind of thing several times, but is it a common thing to use the quantity as the prefix? 
`aVisitor` `someUrls`